app/services/ndelius/api.rb

```diff
@@ -4,10 +4,15 @@ class Api
 
     class << self
       delegate :get_record, to: :instance
+      delegate :get_records, to: :instance
     end
 
     def get_record(nomis_id)
       FakeRecord.generate(nomis_id)
     end
+
+    def get_records(nomis_ids)
+      nomis_ids.map { |id| FakeRecord.generate(id) }
```

Comment:
How will we map the nomis_ids provided to the FakeRecords returned?  
Might it be better to return 

```
{
    "id1": FakeRecord...,
    "id2": FakeRecord...,
}
```

Reply:
Cool - have pushed the change @rossjones